addons/hr/static/tests/m2x_avatar_employee.test.js

```diff
@@ -54,7 +54,7 @@ test("many2one in list view", async () => {
     // click on first employee avatar
     await contains(".o_data_cell .o_m2o_avatar > img:eq(0)").click();
     await waitFor(".o_avatar_card");
-    expect(".o_card_user_infos > span").toHaveText("Mario");
+    expect(".o_card_user_name").toHaveText("Mario");
```

Comment:
There is a lot of noise in the diff due to this change, we could make a prep PR to do this rename.